source/local/local.go

```diff
@@ -45,6 +45,9 @@ const ExpiryTimeKey = "expiry-time"
 // DirectivePrefix defines the prefix of directives that should be parsed
 const DirectivePrefix = "# +"
 
+// MaxFeatureFileSize defines the maximum size of a feature file size
+const MaxFeatureFileSize = 64000
```

Comment:
nit: make this exactly 64kB as advertized

```suggestion
const MaxFeatureFileSize = 65536
```